src/common/xo/index.js

```diff
@@ -972,12 +972,19 @@ export const addAcl = ({subject, object, action}) => (
   call('acl.add', resolveIds({subject, object, action}))::tap(
     subscribeAcls.forceRefresh
   )
-)
+).catch(err => error('Add ACL', err.message || String(err)))
```

Comment:
Or maybe, similarly to `then()`, `tap()` could accept a second param for the rejection case?